### DISCUSSION TITLE
Compliance and Security Operators not available in OKD

### DISCUSSION THREAD

**Discussion by ALL-SPACE-Anas** [okd-project/okd#2046] [Q&A] [answered]
Hi,

I have an OKD4.15 bare-metal cluster. I'm trying to install "Red Hat Quay Container Security Operator" (following this documentation https://docs.okd.io/4.15/security/pod-vulnerability-scan.html#security-pod-scan-cso_pod-vulnerability-scan) and "Compliance Operator" (https://docs.okd.io/4.15/security/compliance_operator/co-management/compliance-operator-installation.html#installing-compliance-operator-web-console_compliance-operator-installation).

I cannot find either of them Searching on the OperatorHub. I've also tried installing them using CLI.

I managed to 'install' Compliance Operator using CLI but I get the below errors so installation isn't complete.
```
constraints not satisfiable: subscription compliance-operator-sub exists, no operators found from catalog redhat-operators in namespace openshift-marketplace referenced by subscription compliance-operator-sub
targeted catalogsource openshift-marketplace/redhat-operators missing
```

For Red Hat Quay Container Security Operator, I cannot proceed with CLI install as I don't have 'container-security-operator' packagemanifest.

- Is there a pre-requisite that I'm missing that doesn't show these operators in the oepratorhub?
- something missing in the documentation?
- My goal is to have a system that can report security issues, package updates, configuration recommendations and vulnerability scans across the cluster (for individual nodes as well as the pods running onto them). If these operators are not installable, please nudge me towards the right options.

Thanks

**Comment by JaimeMagiera** [comment, marked as answer]
The documentation is incorrect. The Red Hat branded operator is not available for non-OCP clusters. OKD needs to include Project Quay Container Security Operator (CSO). We can look at this for 4.16. SCOS.
